librz/core/p/core_plugin_example.c

```diff
@@ -18,23 +18,41 @@ mv core_plugin_example.so ~/.config/rizin/plugins
 #undef RZ_IPI
 #define RZ_IPI static
 
-static int rz_cmd_example_init(void *user /* RzCmd* */, const char *unused /* unused during init */) {
-	RzCmd *cmd = (RzCmd *)user;
-	(void)cmd;
+static const RzCmdDescHelp example_usage = {
+	.summary = "Example core plugin",
+};
+
+RZ_IPI RzCmdStatus rz_cmd_example_handler(RzCore *core, int argc,
+	const char **argv) {
+	if (argc < 2) {
+		return RZ_CMD_STATUS_WRONG_ARGS;
+	}
+	eprintf("This example was called!\n")
```

Comment:
I think you forgot to put a semicolon?

```suggestion
	eprintf("This example was called!\n");
```

Reply:
classic mistake on the first PR :D I will fix it!